InnerEye/ML/dataset/sample.py

```diff
@@ -127,21 +127,19 @@ def get_dict(self) -> Dict[str, Any]:
 class PatientDatasetSource(SampleBase):
     """
     Dataset source locations for channels associated with a given patient in a particular dataset.
+    Please note that "ground_truth_channels" is optional.
```

Comment:
Please clarify. the field is not optional, but it can contain None. Please also describe what those mean.